src/api/forms/form-metadata-repository.js

```diff
@@ -63,6 +63,24 @@ export async function create(document) {
   }
 }
 
+/**
+ * Update a document in the database
+ * @param {string} formId - ID of the form
+ * @param {object} patch - form metadata document
```

Comment:
Ah `Partial` is what I needed. Nice.

Reply:
> I assume we're going to allow any partial update FormMetadataDocument?

Eventually yes maybe, but tailored/controlled through the routes endpoints we offer (i.e. maybe not a single endpoint to "patch"). Not sure yet :thinking: .

Reply:
Only mentioning because we `throw new InvalidFormDefinitionError()` via **createForm()**

Despite having the schema checks at route level too

Depends if this API ever gets exported